cdk/l2-cdk-resources/third-party-integration/yarn.lock

```diff
@@ -0,0 +1,6547 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need this file, if `package.lock` already exists? We've had issues previously with libraries that scan the codebase when both package managers are present.

Single source of truth is best, if possible.

Reply:
removed package-lock